### PR TITLE
Register joltz.is-a.dev

### PR DESCRIPTION
**Pull request by joltz1234**
Register joltz.is-a.dev with CNAME record pointing to https://joltz1234.github.io.

### CODE REVIEW

**Review by Omxpro — APPROVED** [review]

File: domains/mystic titans.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "joltz1234",
+        "email": "joltzx123@gmail.com"
+    },
+    "record": {
+        "URL": "https://discord.gg/nxnsdjb4wm"
```
Comment (wdhdev) [inline domains/mystic titans.json:7]: Invalid URL, 404

File: domains/joltz.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "joltz1234",
+           "email": "75178291+joltz1234@users.noreply.github.com",
```
Comment (wdhdev) [inline domains/joltz.json:4]: Invalid email

**Review by wdhdev — APPROVED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry not approved, please fix the comments above.

**Comment by Stef-00012** [comment]
seems good now

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**